runtime/bcutil/cfreader.c

```diff
@@ -1804,6 +1832,21 @@ checkMethods(J9PortLibrary* portLib, J9CfrClassFile* classfile, U_8* segment, U_
 			}
 		}
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+#if defined(J9VM_OPT_VALHALLA_NEW_FACTORY_METHOD)
+		if (nameIndexOK && utf8Equal(&classfile->constantPool[method->nameIndex], "<new>", 5)) {
+			if (J9_ARE_NO_BITS_SET(value, CFR_ACC_STATIC)) {
+				errorCode = J9NLS_CFR_ERR_INVALID_FLAGS_ON_NEW__ID;
+				goto _errorFound;
+			}
+			if (J9_ARE_ANY_BITS_SET(value, ~CFR_INIT_VT_METHOD_ACCESS_MASK)) {
+				errorCode = J9NLS_CFR_ERR_INVALID_FLAGS_ON_NEW__ID;
+				goto _errorFound;
+			}
+		}
+#endif /* #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
```

Comment:
This is `endif` for `defined(J9VM_OPT_VALHALLA_NEW_FACTORY_METHOD)` and the next line is the `endif` for `defined(J9VM_OPT_VALHALLA_VALUE_TYPES)`. Please fix the comment in this line and in the following line.

Reply:
I see there's a few more places I did this too... I'll fix them all

Reply:
fixed in f4ab7da